src/plugins/curator/services/VideoTeaserSubscriber.js

```diff
@@ -57,6 +57,7 @@ export default class VideoTeaserSubscriber extends EventSubscriber {
 
     if (isCreateForm && target) {
       node
+        .set('image_ref', target.get('image_ref'))
```

Comment:
@rsumilang if we do this in the TeaserSubscriber then all teasers with a target can get this feature.

Reply:
see latest change